### PR TITLE
Create signup page UI

### PR DESCRIPTION
**Pull request by pradeep-suresh2002**
Added UI for Sign up page

### CODE REVIEW

File: Login_demo_application/src/app/login/login.component.html [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-        <input matInput type="password" class="login-page-inputs" placeholder="Password" >
+        <input matInput type="email" class="auth-page-inputs" placeholder="Email id">
+        <input matInput type="password" class="auth-page-inputs" placeholder="Password" >
         <mat-icon class="visibility-icon">visibility</mat-icon>
       </div>
 
       <span class="forgot-password"><u>Forgot password ?</u></span>
       <button class="login-button">Login</button>
-      <span class="nav-signup">Don't have an account? <u class="register">Register</u></span>
+      <span class="nav-signup">Don't have an account? <u class="register" (click)="signUp()">Register</u></span>
```
Comment (Aravindhan-soliton) [inline Login_demo_application/src/app/login/login.component.html]: Give appropriate function name.

File: Login_demo_application/src/app/login/login.component.ts [hunk 2]
```diff
@@ -9,5 +10,15 @@ import { FormControl } from '@angular/forms';
...
   hide = true;
   passwordControl = new FormControl('');
+  constructor(
+    private router : Router
+  ){
+
+  }
+
+  signUp() : void{
+    console.log("In signup")
```
Comment (Aravindhan-soliton) [inline Login_demo_application/src/app/login/login.component.ts]: console.log should not be in PR

File: Login_demo_application/src/app/login/login.component.ts [hunk 3]
```diff
@@ -9,5 +10,15 @@ import { FormControl } from '@angular/forms';
 export class LoginComponent {
   hide = true;
   passwordControl = new FormControl('');
+  constructor(
+    private router : Router
+  ){
+
```
Comment (Aravindhan-soliton) [inline Login_demo_application/src/app/login/login.component.ts:14]: Lint check once

**Review by Aravindhan-soliton — CHANGES_REQUESTED** [review]

File: Login_demo_application/src/app/login/login.component.ts (hunk 3 above)
Comment (pradeep-suresh2002) [reply Login_demo_application/src/app/login/login.component.ts:14]: Checked and updated

File: Login_demo_application/src/app/login/login.component.ts (hunk 2 above)
Comment (pradeep-suresh2002) [reply Login_demo_application/src/app/login/login.component.ts]: Removed

File: Login_demo_application/src/app/login/login.component.html (hunk 1 above)
Comment (pradeep-suresh2002) [reply Login_demo_application/src/app/login/login.component.html]: Updated

**Review by Aravindhan-soliton — APPROVED** [review]

**Pull request merged**